docker-compose-localstack.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  localstack:
+    image: localstack/localstack
```

Comment:
Wow localstack seems awesome! Let's move this into the standard `docker-compose.yml` used for development?